src/modules/dpos/validator/components/LatestVotes/index.test.js

```diff
@@ -1,60 +1,69 @@
-    loadData: jest.fn(),
-    clearData: jest.fn(),
-    urlSearchParams: {},
-    meta: { total: delegateList.length, count: 10, offset: 0 },
-  },
-  t: str => str,
-  delegates: { data: delegateData },
-};
+jest.mock('src/modules/transaction/hooks/queries');
+jest.mock('../../hooks/queries');
 
 describe('Latest votes', () => {
+  const mockFetchNextDelegates = jest.fn();
+  const mockFetchNextTransactions = jest.fn();
+  const props = {
+    filters: {},
+  };
+
   it('displays initial table of votes', () => {
+    useDelegates.mockReturnValue({
+      data: mockDelegates,
+      isFetching: false,
+      fetchNextPage: mockFetchNextDelegates,
+      hasNextPage: false,
+    });
+    useTransactions.mockReturnValue({
+      data: { ...mockTransactions, data: mockTransactions.data.slice(0, 10) },
+      isFetching: false,
+      fetchNextPage: mockFetchNextTransactions,
+      hasNextPage: false,
+    });
     const wrapper = mountWithRouter(LatestVotes, props);
     expect(wrapper.find('.transactions-row').hostNodes()).toHaveLength(10);
   });
 
   it('can load more votes if possible', () => {
-    const updatedDelegatesDataList = delegateList(30);
-    const updatedDelegateData = Object.assign({}, ...updatedDelegatesDataList);
-    const updatedProps = {
-      ...props,
-      votes: {
-        ...props.votes,
-        meta: { total: updatedDelegatesDataList.length, count: 10, offset: 0 },
-      },
-      delgates: { data: updatedDelegateData },
-    };
-    const wrapper = mountWithRouter(LatestVotes, updatedProps);
+    useDelegates.mockReturnValue({
+      data: mockDelegates,
+      isFetching: false,
+      fetchNextPage: mockFetchNextDelegates,
+      hasNextPage: false,
+    });
+    useTransactions.mockReturnValue({
+      data: { ...mockTransactions, data: mockTransactions.data.slice(0, 10) },
+      isFetching: false,
+      fetchNextPage: mockFetchNextTransactions,
+      hasNextPage: true,
+    });
+
+    const wrapper = mountWithRouter(LatestVotes);
     expect(wrapper.find('.load-more')).toExist();
     wrapper.find('.load-more').first().simulate('click');
-    expect(props.votes.loadData).toHaveBeenCalledTimes(1);
+    expect(mockFetchNextTransactions).toHaveBeenCalledTimes(1);
   });
 
   it('can not load more votes if meta property is unavailable', () => {
-    const updatedDelegatesDataList = delegateList(30);
-    const updatedDelegateData = Object.assign({}, ...updatedDelegatesDataList);
-    const updatedProps = {
-      ...props,
-      votes: {
-        ...props.votes,
-        meta: {},
-      },
-      delgates: { data: updatedDelegateData },
-    };
-    delete updatedProps.votes.meta;
-    const wrapper = mountWithRouter(LatestVotes, updatedProps);
+    useDelegates.mockReturnValue({
+      isFetching: false,
+      fetchNextPage: mockFetchNextDelegates,
+      hasNextPage: false,
+    });
+    useTransactions.mockReturnValue({
+      isFetching: false,
+      fetchNextPage: mockFetchNextTransactions,
+    });
```

Comment:
Please don't mock query hooks follow the discussions below
https://github.com/mswjs/msw/discussions/783